files/leaflet.routingmachine/update.json

```diff
@@ -0,0 +1,8 @@
+{
+	"packageManager": "bower",
+	"name": "leaflet.routingmachine",
```

Comment:
`bower                        ENOTFOUND Package leaflet.routingmachine not found`